src/database/DataTypes/TemperatureBreach.js

```diff
@@ -29,9 +38,9 @@ export class TemperatureBreach extends Realm.Object {
   }
 
   get duration() {
-    return moment
-      .duration(moment(this.endTimestamp).diff(moment(this.startTimestamp)))
-      .asMilliseconds();
+    return moment.duration(
+      moment(this.endTimestamp ?? new Date()).diff(moment(this.startTimestamp))
```

Comment:
Changed the behaviour here to use the duration object returned in the utility method created later in the pr